### ISSUE TITLE
wk4 sep_epic - wk1 oct_epic MTL Feature - VA Red Pre/Post Emails w/ Partner, Build, Apply 

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#1176]
Updated description on 2/19:
Need to edit MTL Red's pre/post emails starting with Part 2 and so forth until email # 24 following the Partner, Build, Apply phases.

**Comment by ritahitching** [comment]
@lijenn I will work on the email template while you finish your Epicenter homework.

**Comment by ritahitching** [comment]
working drafts:

**Pre**
https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md

**Post**
https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md

**Comment by lzim** [comment]
@staceypark @dkibbe @branscombj Will everyone review and provide online feedback on the work in progress? Thanks!

**Comment by ritahitching** [comment]
Replace EES logo with TMS logo

**Comment by branscombj** [comment]
My feedback:
- Remove redundancies at the top and bottom of both emails. The header and content for Done should only appear one time; same for Do.
- Correct the Team and Vapor logo distortion.
- Consider truncating the bios & any other long paragraphs of text and instead providing links to more info. This should be an easy glance email.
- Use one section header level only (pre- template shown has two large headers at top followed by 3 medium)
- Make all the graphics and images exactly the same size. I know it's hard in GitHub, but I think it would be easier on the eye if the text could wrap around images.
- Name 3-4 sections that will be in every pre- and post- email. (Might be the same or different for pre vs post.) Could be something like - for pre-email:
DONE/DO - I would put this at the top and put both of them there just as they were at the end of the previous session.
LOGISTICS - this would be where the co-facs would be introduced for S01, as well as other details about dates, technology, etc.
ACCREDITATION - this would have the link to the correct brochure in TMS. They can register before the session and log back in after to take the post-test and evaluation for credit. (note: call them the generic CEU, not CME)
INSPIRATION - this would have the short VAPOR video

**Comment by ritahitching** [comment]
@branscombj thank you for the excellent feedback on the templates.

**Comment by ritahitching** [comment]
@lzim @staceypark any thoughts on making all the graphics and images exactly the same size as per @branscombj suggestions?

**Comment by ritahitching** [comment]
Hi all,
@branscombj thank you for your comments, I incorporated some of your excellent suggestions, if you get a chance please have a look.

@lzim @anthonycpichardo @staceypark @dkibbe @lijenn  a gentle reminder to please review the pre and post email the template [Pre_session_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md) and [Post-session_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)

what does everyone think about @branscombj  suggestion of making all the graphics and images exactly the same size? 
thanks!

**Comment by staceypark** [comment]
@ritahitching there are some typos in the template, including spelling errors and missing punctuation. Please double check.

We should lead with the session topic after the logo, unless EES has requested otherwise. If you look to the existing pre/post templates, they are already there. i.e. "This week we're modeling to learn how to align our team vision" or "We decided on our team vision."

I'm not sure why there are three done/do sections in the post session 01 template, can you explain why? meaning there's an additional section about the "Do" for Week 2, but that should just be part of the pre session 02 email, no?

I don't think any of these headers need to be all caps.

**Comment by lzim** [comment]
@ritahitching 

- Retain Pre/Post email format that we've already set-up and used.
- New information needs to be integrated and the template needs to be formatted for review.

**Comment by ritahitching** [comment]
@lzim 

1. I used the guidance above and recreated a revised working draft of the pre and post email templates. 
2. You can review [Pre_session_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md) and [Post-session_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)
3. It is challenging to make the images the same size, but I tried my best to make them look similar where possible. 
4. The exact sizing suggested above 45" by 75" does not work for the majority of images, but I did use it where possible.
5. I adjusted the Youtube images and the logos to appear consistent.
6. I will also send you a mock up on an email for you to check if you can see the entire message in the preview pane. 
thanks,

**Comment by staceypark** [comment]
@ritahitching I'm looking at the Pre Session 01 template:
1. The session 01 logo is distorted
2 the Do table is not consistent with the one we've already setup and used. The information is split up and not in the order that Lindsey outlined above.

**Comment by ritahitching** [comment]
@staceypark thanks for your help earlier. If you get a chance please have a look at the [template](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md) thanks

**Comment by staceypark** [comment]
@ritahitching the session logo and the VAPOR video thumbnail are touching. They need spacing between them.
The Do table with the session instructions, facilitator info, and accreditation link is disjointed into three blocks? It should be one table.

**Comment by ritahitching** [comment]
@staceypark thanks for your comments. 

1. Tables
- I had made 3 separate tables to match the look of the original [post template emails](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_post_session01.md) where the **do** and **done** are as 2 tables.  
- Should we keep the look consistent for the pre and post? Single table or separate tables for both pre and post?  
- I made the changes  for your review.

2. Facilitators
- The Facilitator photos and bios as a single line doesn't look great as a single line. 
- Not sure, it will display well in the preview pane of outlook. 
- I changed it for your review.

**Comment by staceypark** [comment]
@ritahitching
Tables:
 I'm not sure what you mean by 3 tables. We don't have 3 tables anywhere.

If you look at the original [Session 01 Pre email](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01.md) - there is one table for the Do

In the original [Session 01 Post email](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_post_session01.md), there are two tables - one for the Do and one for the Done.

I'm not sure where you came up with three tables from, but there should be one for the Do and one for the Done.


Facilitator photos:
Photos need to be aligned with each other. In the Pre Session 01, Jane's photo is slightly higher than Matt's. Also, can you make the text aligned to the bottom of the photos, like how the mtl.how/live section and TMS section look.

**Comment by ritahitching** [comment]
@staceypark 
Thanks for feedback. 

I rearranged all the information into 1 table for the [Session 1 template](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md) for pre_post email and aligned the text and images for the Facilitators. 

For the post [Session 1 template](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md) I kept the 2 tables for **Done** and **Do**

I will wait before updating the other session templates

Do you have any guidance on the link for the TMS logo?

Please review when you get a chance. thanks.

**Comment by staceypark** [comment]
@ritahitching Please take a look at the below. I made this in my own branch:
https://github.com/lzim/teampsd/blob/sp_pre_session_01_template/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md

1. All the images are the same size and aligned.
2. You'll notice I didn't workshop the language in the facilitators section, as I'm just showing the formatting. However, I did notice in your facilitators section there isn't a bio at all about them, so please include a short sentence for each.
3. I added a "tab" between the TMS logo and the text that follows since this logo doesn't have enough white space and the text looks attached to the image. You can "tab" in markdown using "&nbsp;"

**Comment by branscombj** [comment]
> @ritahitching Please take a look at the below. I made this in my own branch:
> 
> 
@staceypark et al, This looks super clean and neat - I love it! 

I'm curious about all the not-session-one version. For them, would the post-session email have the Done at the top and the Do at the bottom, and the pre-session just have the Do, at the top? 
Also, the co-fac head-shots are only for S01, right?

Just now jumping back in -  I've followed comments via email pings, but been in long meetings and not had a chance to review developing drafts til now.

**Comment by ritahitching** [comment]
@staceypark 
Thank you for the feedback and the mock up.  
Your table looks even cleaner than mine, and clearly know more markdown than I do! 😄   
I added a short bio for each of the facilitators.  
Any thoughts on the link to EES? Should it be the TMS log in page or the EES brochure page?

@branscombj 
Thanks for your input. 
Yes, from what I understand the facilitators only appear in the first S1 pre_email.
I used the same template for the [post_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md), please have a look.

@branscombj & @staceypark 
**Some pre_template questions:**
1. I placed Matt's bio under his photo and Jane's next to her photo. What do think looks better?  I prefer the bio under the photo. 
2. Should we add a heading: **Your Facilitators for _MTL_**  to the section with the bios before the photos? Perhaps have the sentence immediately under the heading **Your facilitators for _MTL_ will be Matt Moore Ph.D and Jane Branscomb, MPH. Learn more about your facilitators and the rest of our team at mtl.how/team**. followed by the photos and bios. Or keep it as is? 
3. Currently the  **tms** logo and log in details is only in S1 pre-session emails. What do you both think about having the **tms** details on every S1-S12 pre_session email?  My reasoning is in case learners forget or did not get a chance register for CMUs in a previous session and to avoid them having to look through prior emails. 
 
**Post_template questions:**
1. The original email template has the 2nd logo for **do** with the check mark but in the pre_template email we are using the **do** logo with the arrows.  Should we have the same logo across both sessions? If so, which do we want? The **do** with the check mark or with the arrow?

**Comment by staceypark** [comment]
@ritahitching 
For the Done/Do logos, the Done should be the one with the check mark and the Do is the one with the arrows. It looks like there is an error with the logos in the original post session 01 email.

In your post session 01 template, you still have three done/do sections. 
Please see my comment about sticking to the same number of sections we originally have. One Done & one Do section each. 

> @ritahitching
> Tables:
> I'm not sure what you mean by 3 tables. We don't have 3 tables anywhere.
> 
> If you look at the original [Session 01 Pre email](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01.md) - there is one table for the Do
> 
> In the original [Session 01 Post email](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_post_session01.md), there are two tables - one for the Do and one for the Done.
> 
> I'm not sure where you came up with three tables from, but there should be one for the Do and one for the Done.
> 
> Facilitator photos:
> Photos need to be aligned with each other. In the Pre Session 01, Jane's photo is slightly higher than Matt's. Also, can you make the text aligned to the bottom of the photos, like how the mtl.how/live section and TMS section look.
> 
> General Image alignment:
> I bet if you made the image vs. text ratio similar for each of the three sections, it would look more aligned overall.
> 
> Can you just try updating the Pre Session 01 template, until we come to agreement?

**Comment by branscombj** [comment]
> @branscombj @staceypark
> **Some pre_template questions:**
> 
> 3. Currently the  **tms** logo and log in details is only in S1 pre-session emails. What do you both think about having the **tms** details on every S1-S12 pre_session email?  My reasoning is in case learners forget or did not get a chance register for CMUs in a previous session and to avoid them having to look through prior emails.

@ritahitching I definitely think TMS has to be in every email, since they get credit session by session. 

Since there will be a different brochure (and therefore link) for every session for every team, we need to have the specific link for that team and that session in each email, _unless_ we have a page somewhere for each team, with restricted access, that has a table of their sessions and TMS brochure links. If we submit the full series of 12 session dates to TMS up front, we could probably include all the future links in this table and I bet the TMS would block someone from trying to get credit for an event that hasn't happened yet.

This way, we can have the same link in every email, and we can say Find the link here to get CEU for this session and other past sessions you attended (up to x weeks ago).

We may also want to put the TMS link to a launch page for MTL Video brochures.

**Comment by ritahitching** [comment]
@staceypark thanks updated to have 2 sections only.

The **do** and **done**  for the emails need updating.  Will you please confirm the best place to link them to? For the
**Pre_Session Email** 
Is it [here](https://github.com/lzim/mtl/blob/master/blue/session01/s01_learner/mtl_session01_see.md)?
**Post_Session Email**  
The **done to [here](https://github.com/lzim/mtl/blob/master/blue/session01/s01_learner/mtl_session01_see.md) and the **do** [here](https://github.com/lzim/mtl/blob/master/blue/session02/s02_learner/mtl_session02_see.md)?

**Comment by ritahitching** [comment]
@branscombj thanks for your feedback on the **TMS** logo.  
I provisionally added the logo and link to **TMS** to the** **post_session email** for others to see and comment on. 

I'll let others weigh in on the best way to handle the need for customized links in every email for every session and every team. In my opinion, your suggestion of submitting all 12 sessions to TMS to enable us to use the same general link to the TMS login page in every email seems more streamlined.

**Comment by branscombj** [comment]
@JoeBloomquist We're trying to figure the best way to get the correct TMS links to our MTL Live participants. As you may know, the bulk of MTL participants will be attending 12 one-hour sessions over 6 months. After the first or 2nd session we will have worked with them to establish a fixed, recurring appointment like every other Tuesday at 9am Pacific. There is a possibility that something could happen during the 6-month period that would require a shift in one or all of the remaining session dates or times.

Would it be efficient for you and your team if we were to give you up-front (probably after the first session) all the sessions and their dates, time, and facilitators for each team that's going through MTL - and then make changes if and when they arise? We would most likely notify you of any scheduling changes at least 2 weeks in advance.

If you made a change to one or more dates, I trust the link would still stay the same?

Then you could give us a list of all those links once you'd set them up in TMS, and we could give the team a table of all the session dates and links for their series.

This is as opposed to setting up only the next 1-3 sessions at a time in TMS and us communicating the links to the right people over and over.

I am assuming that the TMSystem can have a page set up for a live event that's in the future, and if someone should try to get credit for it the system would block them until after the event date?

Thanks for your thoughts!

**Comment by ritahitching** [comment]
cross reference #1278 to Replace Broken Links in Do and Done Logos in Pre and Post Emails

**Comment by staceypark** [comment]
@ritahitching 
Matt's bio says in the email says that he is a Technical Assistance Specialist - but we know that they are the QIIC (Quality Improvement Implementation Consultant)'s now! 😄 If this needs to be updated for other QIIC's at mtl.how/team as well, please do that (fyi: @lijenn).

See my branch here:
https://github.com/lzim/teampsd/blob/sp_pre_session_01_template/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md

I also added a "&nbsp;" space between the MTL session logo and the VAPOR video so they wouldn't be touching. And I reduced the bios to be one-line for a cleaner look.

Let me know what you guys think!

**Comment by lzim** [comment]
Hi @ritahitching @staceypark and @lijenn 

I think we are getting on the right track 🥇 !

If this is the latest iteration on "sp_pre_session_01" branch: https://github.com/lzim/teampsd/blob/sp_pre_session_01_template/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md

1. I don't under stand the lead sentence: "They will be your facilitators for MTL."
- This doesn't make sense, or it is out-of-context and doesn't italicize _MTL_...
- Does it go in another location?

2. The TMS logo, photos and the small MTL logos are all fuzzy??

**Comment by staceypark** [comment]
@lzim WHOOPs I just got what you meant by the "They will be your facilitators..." sentence.
Just deleted that. I copied that over from an earlier version so I could edit it and put it in the body.

The facilitator section of the table now says "Introducing your _Modeling to Learn_ Co-Facilitators".

@ritahitching or @lijenn can one of you investigate the fuzzy logos

**Comment by branscombj** [comment]
Hi @lijenn @ritahitching @staceypark @lzim  

- The small mtl logo doesn't look fuzzy to me; the TMS one might be a tad fuzzy - or it could be my eyeglass Rx needs updating but will have to do for another few months.
- I wonder if the Do icon could be fashioned in the same way as the others in the email: on the left, not in a markdown table - OR - have it look just as it does in the guides, in a side-by-side table without a separate line saying "at our next team meeting".
- I was promoted a couple months ago :) my new title is Assistant Project Director - could you please correct this at mtl.how/teams and wherever?
Thanks!

**Comment by branscombj** [comment]
Hi @ritahitching @lijenn @staceypark @lzim 
Based on Elizabeth's reply via email to my questions above, it sounds like when we launch a team, we'll give EES the information for the S01 brochure, and then after S01 - or S02 at the latest - give them the info to upload all the rest of the brochures. Then we can provide each team with a set of links for each of their sessions - perhaps in something like the table attached.

-> Where would that table live? I know we have mtl.how/tms currently linked to tms. Would we instead have it link to a page that lists all the current teams and links each one to a page with the information below? (seems clunky) Or do we only give participants the links in their emails?

This table should probably include video links, as shown, to be handy for makeups.
[teamclassandtmslinkstable.xlsx](https://github.com/lzim/teampsd/files/4428378/teamclassandtmslinkstable.xlsx)

Of course we need to think through *MTL* Red versions for this as well as the emails themselves.

**Comment by ritahitching** [comment]
Hi @branscombj 
Congratulations on your promotion! 🥇 
@lijenn is updating the mtl.how/team page to reflect the tittle changes for your role and the remainder of the facilitators to QIICS #1279 

- Thanks for contacting EES to get additional guidance and for putting the table together. I'll let @staceypark @lzim weigh in on what's the best way to direct learners to TMS.  The logo currently links to [VA Course List](https://www.docs.lms.va.gov/LMSDocs/default.aspx?id=56771) not the main [TMS Login](https://www.tms.va.gov/SecureAuth35/). I'll update links once we decided. 
- I **replaced** the previous TMS logo - I hope this one is clearer.
- I aligned the **do** logo to the left of the table for your review.

Updated templates here for  **[pre_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md)** and **[post_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)**  

thanks all

**Comment by branscombj** [comment]
@ritahitching I like it with the Do on the left! See what others think.
In the post email the Done logo is a bit larger than the others and not all the way to the left.
Thank you

**Comment by ritahitching** [comment]
> @ritahitching I like it with the Do on the left! See what others think.
> In the post email the Done logo is a bit larger than the others and not all the way to the left.
> Thank you

@branscombj glad you liked it.  I made the **done** logo smaller and left aligned further. thanks!

**Comment by lzim** [comment]
@ritahitching 
1. Edit the link "TMS Intranet." [hyperlink intranet] "TMS Internet" [hyper internet]
2. Add any facilitator photos with the circle to a new "small_circle_headshots" folder on Team PSD Resources folder and then we can try & get them higher res down the road.

**Comment by lzim** [comment]
@ritahitching and @branscombj 
- This back-and-forth is getting hard to follow, which parts are done, and which are still open questions. 
- I'm trying to list out each requirement/decision or open question in a checklist format, providing my answer/recommendation.

From @ritahitching 

> - [ ] @lijenn is updating the mtl.how/team page to reflect the tittle changes for your role and the remainder of the facilitators to QIICS #1279

From @branscombj 
> Based on Elizabeth's reply via email to my questions above, it sounds like when we launch a team, we'll give EES the information for the S01 brochure, and then after S01 - or S02 at the latest - give them the info to upload all the rest of the brochures. Then we can provide each team with a set of links for each of their sessions - perhaps in something like the table attached.

- I like your table attached. I do wonder about having it in any easy to check location online.

From @ritahitching 
> Thanks for contacting EES to get additional guidance and for putting the table together. I'll let @staceypark @lzim weigh in on what's the best way to direct learners to TMS. The logo currently links to VA Course List not the main TMS Login. I'll update links once we decided.
- [x] We'll link to "TMS Intranet." "TMS Internet."

@branscombj 
> -> Where would that table live?

- I'm very open to ideas about where the table would live. 

> I know we have mtl.how/tms currently linked to tms. 

- See my "Intranet" and "Internet" comment above.

> Would we instead have it link to a page that lists all the current teams and links each one to a page with the information below? (seems clunky) 

> Or do we only give participants the links in their emails?

- I'd like to recommend that we do both: emails and a webpage that they can easily check.

@branscombj 
> This table should probably include video links, as shown, to be handy for makeups.
teamclassandtmslinkstable.xlsx
- [ ] It will be nice to create a public-facing place for this table, so that it is accessible and easy.

> Of course we need to think through MTL Red versions for this as well as the emails themselves.
- [ ] I'm not sure I understand what you mean here, @branscombj

**Comment by branscombj** [comment]
> Of course we need to think through MTL Red versions for this as well as the emails themselves.

- [ ] I'm not sure I understand what you mean here Jane

@lzim There are unique aspects to Red vs Blue that will require a little bit of adapting for both the table of sessions/links I proposed and for the pre-/post-emails themselves that are being discussed in this thread. Just want to be sure that if we're creating templates, we cover the variations. (Same issue with needing a mock-up of a session after S01, since that one has end-case uniqueness.)

E.g. The table for date/session/Guide/Video/TMSlink in Red will look more like date/part (1 six times, 2 six times)/guide (1 six times, 2 six times)/TMSlink.

The pre-/post emails for Red might not be too different from those for Blue, except they'll only come in two variations, 6 all looking mostly the same, with the same heading and Done/Do, and then the other 6 looking the same. Internally we'd probably number them 1-12 and use the same 12 VAPOR video clips and some other content by session, like Your Facilitators for #1 of Red Part 1.

**Comment by ritahitching** [comment]
Check consistency with the requirements  in document_tracker issue card #987

**Comment by lzim** [comment]
> Check consistency with the requirements in document_tracker issue card #987

Thanks for documenting this during the #hqhuddle @ritahitching 🔥 👍

**Comment by ritahitching** [comment]
Updates:
1. Updates made to the document_tracker #987 
2. Decisions: https://www.tms.va.gov/SecureAuth35/ for CMUs with EES 
3. I'll update the template to reflect this decision

**Comment by ritahitching** [comment]
@ritahitching 
Update pre and post email template links once pullrequest #1285 has been made
- [ ] links to facilitators photos
- [ ] links to vapor team youtube screenshots

**Comment by ritahitching** [comment]
@staceypark 
- [ ] Does it make sense to add this issue for triage at the HQ meeting 4/9, so that we decide on the template and start working on the 24 emails for MTL blue?
- [ ] If so, I'll update the agenda 
thanks

**Comment by ritahitching** [comment]
- [ ] Update link to TMS logos in pre and post email template once icons have been merged to master - tracked in branch #1287 and branch rita_2020_04_08_email_template_resources_images_icons

**Comment by ritahitching** [comment]
@lzim Will you please
- [ ] review the pre and post MTL Blue email templates and provide any final edits - details  for **[pre_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md)** and **[post_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)**  
- [ ] confirm if I may go ahead and create the remaining S2 to S12 emails templates or the pre and post sessions.

thanks

**Comment by lzim** [comment]
@ritahitching 

I've reviewed the **[pre](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md)** and **[post](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)**  

- _What are the decision rules for the use of **bold** text?_ The example run into the problem: "If everything is bold, then nothing is bold." It's especially acute in the **post** example, where almost nothing standards out.

- _How will the dates in the YAML header ultimately be handled?_

FYI: @lijenn @staceypark

**Comment by ritahitching** [comment]
@lzim 
- [ ] Thanks for reviewing the [pre](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md) and [post](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)
- [ ] I don't know what the recommendation is for the use of the **bold** in the pre and post email template.  Generally we only want to **bold** the most salient part(s).   
- [ ] I kept the same sections that were **bold** in the [original pre template](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01.md)  and [original post template](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_post_session01.md) where almost everything was **bolded**.
- [ ] I don't know how the YAML headers will be handled.  I was thinking they would not be included in the email as they were being sent from outlook.

**Comment by lzim** [comment]
> @lzim
> 
> * [ ]  Thanks for reviewing the [pre](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md) and [post](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)
> * [ ]  I don't know what the recommendation is for the use of the **bold** in the pre and post email template.  Generally we only want to **bold** the most salient part(s).
> * [ ]  I kept the same sections that were **bold** in the [original pre template](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01.md)  and [original post template](https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_post_session01.md) where almost everything was **bolded**.
> * [ ]  I don't know how the YAML headers will be handled.  I was thinking they would not be included in the email as they were being sent from outlook.

@ritahitching I look forward to you turning all of your "I don't knows" into well-considered and documented recommendations for consideration by the team on this issue, so that #1176 can move forward.

FYI: @lijenn

**Comment by ritahitching** [comment]
@lzim Please see my recommendations below. 

## **Recommendations**

- [ ] [Pre_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/pre_emails/mtl_live_pre_session01_template.md) and [post_email](https://github.com/lzim/teampsd/blob/rita_2020_03_19_email_template_issue_1176/mtl_facilitate_workgroup/pre_post_emails/post_emails/mtl_live_pre_session01_template.md)

### **Bolding**
- [ ] Using the guidance for [emails](https://github.com/lzim/teampsd/blob/master/resources/training_guides/teampsd_emails.md) and [mtl_principles](https://github.com/lzim/teampsd/blob/master/resources/training_guides/teampsd_guiding_principles.md)
- [ ] I would recommend only **bolding** the **At our next meeting**, the **date** and **phone number** details 
- [ ] The section headers are set to a larger font using ## - making them appear bolded -  I would suggest reducing the size for all section headers

### **YAML Headers**
- [ ] Suggest removal of headers
- [ ] Mock up on outlook
![image](https://user-images.githubusercontent.com/54862187/79608685-49d96100-80aa-11ea-9583-b1abb252041e.png)

![image](https://user-images.githubusercontent.com/54862187/79608554-14cd0e80-80aa-11ea-81b1-b897b7435569.png)

**Comment by lzim** [comment]
Thanks @ritahitching 👏 

@branscombj @dlkibbe and @lijenn - Please note that this is not a priority for this week, as it's not on the **april_epic** and there are other more pressing priorities.

- But, will **facilitate_workgroup** folks please take the first step of reviewing Rita's recommendations regarding use of **bold**. The last version I reviewed had all bold, which meant nothing was bold.

@ritahitching I think it's okay to **hide** the YAML header in the .html email file. But, we won't want to actually remove the YAML header from the .md file, as it determines how the output file will be generated and a variety of other things.
See: https://cran.r-project.org/web/packages/ymlthis/vignettes/yaml-fieldguide.html

**Comment by branscombj** [comment]
@ritahitching @lzim @dlkibbe 
> I would recommend only bolding the **At our next meeting**, the **date** and **phone number** details.  
> The section headers are set to a larger font using ## - making them appear bolded - I would suggest reducing the size for all section headers.

I agree with both of these. Consistent font size is easier to take in; and only bold the section headers. That goes for the content under At our Next MEeting too - do not bold steps 1 and 2 about login and VANTS. The  important thing is to point the reader's eye to where the key information is, then let them read it in plain font.

**Comment by ritahitching** [comment]
@ritahitching Keep YAML headers  in .md needed to format documents but remove from .html

**Comment by lzim** [comment]
> @ritahitching Keep YAML headers in .md needed to format documents but remove from .html

@ritahitching  I just saw you last comment, and I’m just making sure that you do NOT “remove” the YAML from .md files, but simply identify the correct YAML code to **hide** the YAML header when they are knit as .html

> @ritahitching I think it's okay to hide the YAML header in the .html email file. But, we won't want to actually remove the YAML header from the .md file, as it determines how the output file will be generated and a variety of other things.

**Comment by ritahitching** [comment]
@lijenn this is issue is blocked by the #1192 and the MVP process.
If you read the thread you will find inks to the most current drafts of the email templates.
I recommend we keep this issue open.

**Comment by lzim** [comment]
@lijenn @anthonycpichardo 
- If this is blocked re: Team PSD manual #1192, can you help make sure it is appropriate indicated as such using ZenHub "blocking?"

**Comment by lzim** [comment]
@lijenn What about these emails?

Are we waiting for the GitHub branches to be set up to edit these? Or is this about waiting to have wave 1 details to fill in? Or, something else?

**Comment by lijenn** [comment]
@lzim I believe both, more specifically I'm thinking of these 3 issues:
- #841 wk3 team_tracker set up: we wanted to move these to the team_tracker repo and the images/links will all need to be updated due to the move
- #1502 wk4 (I'm going to change this to week 3 since it's a do-able 3 pointer) documentation across do/done tables: checking for consistency in language and update that in the pre/post emails
- #983 wk3: wave1 details to fill in

**Comment by staceypark** [comment]
@lzim  @lijenn we are hoping these will go directly in the CRM so we may not need this task.

**Comment by lzim** [comment]
@staceypark @lijenn and @anthonycpichardo 
Where does this fit with CRM MVP or pre-MVP?

I still can't tell whether/how we are approaching this across #841, #1502, and #981.

It seems #1502 and #981 are needed in the CRM MVP, whereas #841 is our back-up if we can't get our co-planning MVP in January and follow-up with team tracking in the CRM in February.

**Comment by lijenn** [comment]
@lzim @staceypark 
I believe any emails needing templates and routing/mapping is included in the January co-planning MVP, is that a correct assumption @anthonycpichardo?

Wouldn't the co-planning MVP would have the same set-up/configuration needs that would cover the pre/post emails since the pre/post emails are just templates that have the same tailoring and mapping functions needed for the co-planning emails?

In addition to #841 team tracker as the back-up for team tracking, I believe it was also #1329 and #1299 as a back-up for email management. 

Whereas, #1502 is just a discrepancy check between the do/done tables from the pre/post emails and the see/say guides.
I believe #983 would be a part of the templated emails MVP?

**Comment by lijenn** [comment]
wk4 feb_epic - On Monday we need to send wave 1 NIH MTL Red to Fresno.

wk1 mar_epic - Post for Fresno.

wk2 mar_epic - Split to second card. Need everyone else for their week 1 meeting.
Pre-email is week of team meeting.

**Comment by staceypark** [comment]
3/15 #mm meeting: 
Decisions - 
1. We will try to keep the content the same across both trials (NIH & VA), but we will need two sets of email templates since NIH will say pre/post for emails 1-4 followed by the week number for emails 5-24 whereas VA will say pre/post for all 24 emails

2. We want to tie in the "DO" each week to Parnter/Build/Apply themes

- [ ] @dlkibbe will look for content from VAPOR videos from each week to tie into the ask for that week
- [ ]

**Comment by branscombj** [comment]
> 3/15 #mm meeting:
> Decisions -
> 
Friendly amendments: 
* Each week's email will incorporate the associated VAPOR video and a "DO" recommendation. 
* The DO will be in the form of Partner/Build/Apply and will tie in to the VAPOR video however possible.
*We may want to flag items from emails 1-4/Parts 1&2 to bring in at key points in the email sequence. E.g. have them circle back to the things they chose to focus in MTL Red Live in the DO for emails 8, 16, 24 or 12 & 24 (corresponding to when the Team Time reports go out for Blue or VA Red).
* Wordsmithing the Partner/Build/Apply language...

**PARTNER** with your team to review a data point or trend at mtl.how/data.
_- specific recommendation for where/what to choose, tied to VAPOR video theme_

**BUILD** your understanding of this data point or trend.
_- further prompts relevant to the data recommendation to get at the why of it_

**APPLY** your understanding to decisions in your team.
_- prompts about what the data leads them to think about decisions they have, could, or will make in their team to better achieve the results they want/meet their priority needs_

**Comment by lzim** [comment]
**Thank you for your work on these!** :heart:

I think you've achieved, the instructions we discussed:
1. **Parallel** _MTL Blue_ partner, build, apply.
2. But, without the **systems thinking**.

But, I believe they are **missing our** focus on:
1. **Veterans** 
2. **Clinical Decision-making**

That we are really hoping the data and facilitation will support.

_I recommend working with these tweaks below:_

**PARTNER with your team to review team trends and your Veterans' care at mtl.how/data.**

**BUILD your understanding of how the data can inform your team's and your own clinical care.**

**APPLY your understanding to decisions in your team.**

@lijenn 
@staceypark 

If you have questions about this, we can cover at **hq_workgroup** this afternoon.

**Comment by lzim** [comment]
@lijenn 

Do these need further revision? Or are these ready to go?

**Comment by staceypark** [comment]
8am workgroup leads 9/13:
- Done with Partner/Build/Apply Learning Objectives

Still need:
- [ ] Done/Do
- [ ] "We decided..."

**Comment by lijenn** [comment]
Now tracked in #2142, this is an outdated card and new effort is tracked in the feature card.
